catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -361,12 +385,7 @@
                             ddf-security-common,
                             platform-util,
                             platform-util-unavailableurls,
-                            failsafe,
-                            hazelcast,
-                            sardine,
-                            httpclient,
-                            httpcore,
-                            commons-lang3
+                            sardine
```

Comment:
❓ Were these embeds not needed anymore?

Reply:
Yes. The bundles are now started as part of the feature instead.